src/frontend/src/helpers/Login.ts

```diff
@@ -0,0 +1,59 @@
+import * as $ from "jquery";
+export {Login, LoginError, Connection}
+
+class LoginError extends Error {
+  constructor(msg: string,
+              public username?: string,
+              public status?: number,
+              public statusText?: number) {
+    super(msg);
+  }
+}
+
+class Login {
```

Comment:
Actually, I remember why I did this..
I wanted the ability to login with other uri other than `https://www.student.cs.uwaterloo.ca/~cs136/seashell/cgi-bin/login2.cgi`,
maybe to `seashell-dev`.